site/docs/v1/tech/apis/identity-providers/_samlv2-request-body.adoc

```diff
@@ -1,43 +1,72 @@
 Determines if this identity provider is enabled for the Application specified by the `applicationId` key.
 
+ifndef::samlv2_idp_initiated[]
 [field]#identityProvider.buttonImageURL# [type]#[String]# [optional]#Optional#::
 The top-level button image (URL) to use on the FusionAuth login page for this Identity Provider.
+endif::[]
 
+ifndef::samlv2_idp_initiated[]
 [field]#identityProvider.buttonText# [type]#[String]# [required]#Required#::
 The top-level button text to use on the FusionAuth login page for this Identity Provider.
+endif::[]
 
+ifndef::samlv2_idp_initiated[]
 [field]#identityProvider.debug# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.7.3#::
 Determines if debug is enabled for this provider. When enabled, each time this provider is invoked to reconcile a login an Event Log will be created.
 +
 This is useful for debugging purposes, but is not intended to be left on during production. It should be enabled during integration or if you are experiencing an issue with this identity provider.
+endif::[]
+ifdef::samlv2_idp_initiated[]
+[field]#identityProvider.debug# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`#::
+Determines if debug is enabled for this provider. When enabled, each time this provider is invoked to reconcile a login an Event Log will be created.
++
+This is useful for debugging purposes, but is not intended to be left on during production. It should be enabled during integration or if you are experiencing an issue with this identity provider.
+// no 'since' needed
+endif::[]
 
+ifndef::samlv2_idp_initiated[]
 [field]#identityProvider.domains# [type]#[Array<String>]# [optional]#Optional#::
 This is an optional list of domains that this SAML v2 provider should be used for. This converts the FusionAuth login form to a domain-based login form. This type of form first asks the user for their email. FusionAuth then uses their email to determine if an identity provider should be used. If an identity provider should be used, the browser is redirected to the login URL of that identity provider. Otherwise, the password field is revealed on the form so that the user can login using FusionAuth.
+endif::[]
 
 [field]#identityProvider.emailClaim# [type]#[String]# [optional]#Optional#::
 The name of the email claim (Attribute in the `Assertion` element) in the SAML response that FusionAuth uses to uniquely identity the user. If this is not set, the `useNameIdForEmail` flag must be true.
 
 [field]#identityProvider.enabled# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`#::
 Determines if this provider is enabled. If it is false then it will be disabled globally.
 
+ifndef::samlv2_idp_initiated[]
 [field]#identityProvider.idpEndpoint# [type]#[String]# [optional]#Optional#::
 The SAML v2 login page of the identity provider.
+endif::[]
+
+ifdef::samlv2_idp_initiated[]
+[field]#identityProvider.issuer# [type]#[String]# [required]#Required#::
```

Comment:
Do we have a flag that indicates `📣 Breaking Change - was required to be a URL until 1.27.1`(or something like that 🙂 )?

Probably not, but just checking.

Reply:
No, we don't have that., and I'm not sure that is a breaking change. Going the other way would be, however (from requiring it to be a string to being a URL).